src/views/_proxied-dot-io/vault-docs-landing/components/use-case-card/style.module.css

```diff
@@ -0,0 +1,16 @@
+.heading {
+  composes: g-type-display-5 from global;
+  margin: 0 !important;
+}
+
+.body {
+  margin: 8px 0 0 0 !important;
```

Comment:
Are the `!important` necessary here / is there a way we can make them not necessary?